bot/exts/moderation/infraction/management.py

```diff
@@ -295,13 +295,7 @@ async def cog_check(self, ctx: Context) -> bool:
         """Only allow moderators inside moderator channels to invoke the commands in this cog."""
         checks = [
             await commands.has_any_role(*constants.MODERATION_ROLES).predicate(ctx),
-            in_whitelist_check(
-                ctx,
-                channels=constants.MODERATION_CHANNELS,
-                categories=[constants.Categories.modmail],
-                redirect=None,
-                fail_silently=True,
-            )
+            is_mod_channel(ctx.channel)
         ]
         return all(checks)
```

Comment:
Not part of this PR's diff, but I feel like we could make the first predicate simpler. Checking the intersection of `ctx.author`'s role ids and the constants should work, unless I'm missing something.

Reply:
I checked the categories, the ID's are correct.
Added the "closes" part

> I feel like we could make the first predicate simpler

Unless there's a matter of efficiency, I think the current line makes it pretty clear what it does.